ddtrace/tracer/remote_config.go

```diff
@@ -298,13 +299,26 @@ func initalizeDynamicInstrumentationRemoteConfigState() {
 			time.Sleep(time.Second * 5)
 			diRCState.Lock()
 			for _, v := range diRCState.state {
+				accessStringsToMitigatePageFault(v.runtimeID, v.configPath, v.configContent)
 				passProbeConfiguration(v.runtimeID, v.configPath, v.configContent)
 			}
 			diRCState.Unlock()
 		}
 	}()
 }
 
+func accessStringsToMitigatePageFault(strs ...string) {
+	for i := range strs {
+		pageSize := os.Getpagesize()
+		for offset := 0; offset < len(strs[i]); offset += pageSize {
+			_ = strs[i][offset]
```

Comment:
Not sure how `go` handles out of bound reads. Could this cause a crash?

Reply:
We're using range to ensure we're not out of bounds, am I missing a mistake here?

Reply:
I was referring to the inner loop, but nevermind, I misunderstood the code.